layouts/partials/location/address.njk

```diff
@@ -0,0 +1,29 @@
+<p property="schema:address" typeof="schema:PostalAddress">
+    <span property="schema:streetAddress">
+      {{address.line1}}
+      {% if address.line2 %}
+        <br>
+        {{address.line2}}
+      {% endif %}
+      {% if address.line3 %}
+        <br>
+        {{address.line3}}
+      {% endif %}
+    </span>
+    <br>
+
+    {% if postcodePosition == "below" %}
+      <span property="schema:addressLocality">{{city}}</span>
+      <br>
+    {% endif %}
+    {% if postcodePosition == "after" %}
+      <span property="schema:addressLocality">{{city}}</span>
+    {% endif %}
+    {% if postRegion %}<span property="schema:addressRegion">{{postRegion}}</span> {% endif %}<span property="schema:postalCode">{{postcode}}</span>
+    {% if postcodePosition == "before" %}
+      <span property="schema:addressLocality">{{city}}</span>
+    {% endif %}
+
+    <br>
```

Comment:
Should we be using `br` tags?

Reply:
In this case, yes. The formatting of the postal address is important here, so `<br>` is a valid way of achieving that.